zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/CamundaExporter.java

```diff
@@ -115,6 +120,7 @@ public void open(final Controller controller) {
     writer = createBatchWriter();
 
     scheduleDelayedFlush();
+    scheduleImportersCompletedCheck();
```

Comment:
❌ We should also have a direct check, otherwise we have a delay of schedule interval.